test/jdk/java/awt/regtesthelpers/PassFailJFrame.java

```diff
@@ -184,6 +262,53 @@ public void windowClosing(WindowEvent e) {
         windowList.add(frame);
     }
 
+    private static void captureScreen(Rectangle bounds) {
+        if (robot == null) {
+            try {
+                robot = new Robot();
+            } catch (AWTException e) {
+                String errorMsg = "Failed to create the " +
+                        "instance of Robot.";
+                JOptionPane.showMessageDialog(frame, errorMsg, "Failed",
+                        JOptionPane.ERROR_MESSAGE);
+                forceFail(errorMsg + e.getMessage());
+            }
+        }
+        MultiResolutionImage multiResolutionImage = robot
+                .createMultiResolutionScreenCapture(bounds);
+        List<Image> imageList =
+                multiResolutionImage.getResolutionVariants();
+        Image image = imageList.get(imageList.size() - 1);
+        File file = new java.io.File("CaptureScreen_" +
+                imgCounter.incrementAndGet() + ".png");
+        try {
+            ImageIO.write((RenderedImage) image, "png", file);
+        } catch (IOException e) {
+            throw new RuntimeException(e.getMessage());
+        }
+    }
+
+    private static void captureScreen(int selectedIndex) {
+        if (selectedIndex == 0) {
+            Arrays.stream(GraphicsEnvironment.getLocalGraphicsEnvironment()
+                    .getScreenDevices())
+                    .map(GraphicsDevice::getDefaultConfiguration)
+                    .map(GraphicsConfiguration::getBounds)
+                    .forEach(PassFailJFrame::captureScreen);
+        }
+
+        if (selectedIndex == 1) {
+            windowList.stream()
+                    .filter(Window::isShowing)
+                    .map(Window::getBounds)
+                    .forEach(PassFailJFrame::captureScreen);
```

Comment:
```suggestion
            windowList.stream()
                      .filter(Window::isShowing)
                      .map(Window::getBounds)
                      .forEach(PassFailJFrame::captureScreen);
```
Align the dots for wrapped calls.